assets/l10n/app_en.arb

```diff
@@ -932,6 +932,24 @@
   "@errorMarkAsUnreadFailedTitle": {
     "description": "Error title when mark as unread action failed."
   },
+  "markAllAsReadConfirmationDialogTitle": "Mark messages as read?",
+  "@markAllAsReadConfirmationDialogTitle": {
+    "description": "Title of the confirmation dialog for marking all messages as read"
+  },
+  "markAllAsReadConfirmationDialogMessage": "{count} messages will be marked as read.",
+  "@markAllAsReadConfirmationDialogMessage": {
+    "description": "Message that warns the user they may unintentionally mark messages as read",
```

Comment:
nit: how about just "Message in the confirmation dialog for marking all messages as read."